<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondwindow</class>
 <widget class="QDialog" name="secondwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 2, 2);</string>
  </property>
  <widget class="QLabel" name="text_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>591</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shoot_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 25pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>shoot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inventory_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 25pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>inventory</string>
   </property>
  </widget>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>590</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hp_label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;PLAYER HP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="my_hp_lcd">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="hp_label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;ENEMY HP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="enemy_hp_lcd">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cookie_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>cookie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="critical_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>500</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>critical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insight_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>530</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>insight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flash_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>flash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reload_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 6, 6);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>reload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="turn_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(129, 61, 156);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>turn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image_comfil/russian_rullet_image.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image_comfil/russian_rullet_image.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gunlabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image_comfil/revolver.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
